test/integration/test_report.py

```diff
@@ -42,12 +42,13 @@ def test_report_granular(builddir):
             "report",
             _path + ":function1",
             "cyclomatic.complexity",
+            "--message",
             "-n",
             1,
         ],
     )
     assert result.exit_code == 0, result.stdout
-    assert "Not found" not in result.stdout
+    assert "not found" not in result.stdout
```

Comment:
This is the only test where you changed `"Not found"` to `"not found"`. Is it intentional to only change this check?